README.md

```diff
@@ -244,9 +244,13 @@ documentation][doc-logging] for more information.
 
 ## Compatibility
 
-- Apache Cassandra versions 2.1 and above.
-- DataStax Enterprise versions 4.8 and above.
-- Node.js versions 8 and above.
+The driver supports all versions of Node.js, Cassandra, and DSE that are not EOL at the time of release.  Only LTS eligible branches (i.e. even numbered releases) are supported for Node.js.  See the [project documentation] for more information about the Node.js release cycle.
+
+The current version of the driver offers support consistent with this policy for the following:
+
+- Apache Cassandra versions 3.0 and above.
+- DataStax Enterprise versions 5.1 and 6.8.
+- Node.js versions 18.x and 20.x.
```

Comment:
@emeliawilkinson24 This preserves the content changes you recommended... I just moved a few of the sentences together into a single paragraph since all the points seemed to be on related content and it reads a bit more cleanly this way.

Just wanted to make sure you're good with what's here.